### PR TITLE
osu-lazer: update to 2020.1031.0

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

Update `osu-lazer` to 2020.1031.0

Package(s) Affected
-------------------

`osu-lazer`

Security Update?
----------------

No

Architectural Progress
----------------------

- [x] AMD64 `amd64`   

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`

### CODE REVIEW

**Review by eatradish — CHANGES_REQUESTED** [review]
Please use `SRCS` and `CHKSUMS`

File: extra-games/osu-lazer/spec [hunk 1]
```diff
@@ -1,3 +1,3 @@
-VER=2020.903.0
+VER=2020.1031.0
 SRCTBL="https://github.com/ppy/osu/archive/$VER.tar.gz"
```
Comment (MingcongBai) [inline extra-games/osu-lazer/spec]: Please use [new ACBS spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/)

**Review by eatradish — APPROVED** [review]

**Review by MingcongBai — APPROVED** [review]

**Comment by OriginCode** [comment]
Works fine on amd64:
image

**Review by cthbleachbit — APPROVED** [review]
LGTM.

**Pull request merged**